src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -684,6 +685,13 @@ void FancyZones::OnDisplayChange(DisplayChangeType changeType) noexcept
 {
     Logger::info(L"Display changed, type: {}", changeType);
 
+    if (changeType == DisplayChangeType::Initialization ||
+        changeType == DisplayChangeType::DisplayChange ||
```

Comment:
is it really needed on DisplayChange and WorkArea? Device ids should remain the same on resolution and workarea changes?

Reply:
hm, if e.g. unplugging monitor causes resolution change than it's ok

Reply:
Agreed with WorkArea, we don't really need it here. DisplayChanged is called when the configuration changes, so we need to keep it here.